modules/DSA-02_project-1.yml

```diff
@@ -26,13 +26,13 @@ screens:
               id: DSA-02-new-content-verification
               success-message: "Great job - you successfully updated the file."
               failure-message: "It looks like you haven't updated the file yet."
-        - description: "Make sure the change shows up under Pull Request #5. If it does, make a comment in the PR about the update."
+        - description: "Create a Pull Request to merge the `add-gitignore` branch into `master`."
           id: DSA-02-pr-comment
```

Comment:
@1point618 dumb question - does this id need to change based on the different verification required here (a comment vs. a new PR)? I would guess the id is irrelevant to the actual verification, but wanted to double check.